test/Cardano/FeeSpec.hs

```diff
@@ -677,7 +731,7 @@ instance Arbitrary FeeOptions where
             }
 
 instance Arbitrary a => Arbitrary (NonEmpty a) where
-    arbitrary = (:|) <$> arbitrary <*> arbitrary
+    arbitrary = (:|) <$> arbitrary <*> (take 10 <$> arbitrary)
```

Comment:
why not use : 
```
num <- choose (1,100) 
```
and use instead 10?

Reply:
Fixed in https://github.com/input-output-hk/cardano-coin-selection/pull/26/commits/efcc8d7019c6a19e3a5d216002ac047208a3c115!